utils/pid.go

```diff
@@ -0,0 +1,96 @@
+package utils
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"runtime"
+	"strconv"
+	"syscall"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// SetProcessIdToFile obtains the process id and save it inside a file
+func SetProcessIdToFile(pidFileName string) error {
+	log.Debug("SetProcessIdToFile")
+
+	pidValue := os.Getpid()
+	log.Debug("current pid:", pidValue)
+	if err := ioutil.WriteFile(pidFileName, []byte(strconv.Itoa(pidValue)), 0600); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// GetProcessIdFromFile reads the process id previously stored in the file
+func GetProcessIdFromFile(pidFileName string) (int, error) {
+	log.Debug("GetProcessIdFromFile")
+
+	pid := int64(0)
```

Comment:
It would be more idiomatic to use `var pid int64`, which is automatically initialized to the int64 zero value